app/views/DraftNoLongerAvailableView.scala.html

```diff
@@ -0,0 +1,48 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.html.components.Heading
+@import config.FrontendAppConfig
+@import views.html.components.Button
+
+@this(
+        mainTemplate: MainTemplate,
+        heading: Heading,
+        button: Button,
+        config: FrontendAppConfig
+)
+
+@()(implicit request: Request[_], messages: Messages)
+
+@mainTemplate(
+    title = messages("draftNoLongerAvailable.title"),
+    showBackLink = true
+) {
+    @heading(messages("draftNoLongerAvailable.heading"))
+
+    <p class="govuk-body">@messages("draftNoLongerAvailable.paragraph")</p>
+
+    <p class="govuk-body">
+        <a class="govuk-link" id="make-new-departure" href="@routes.RedirectController.onPageLoad()">@messages("draftNoLongerAvailable.newDeparture")</a>
+    </p>
+
+    <div>
+        @button(
+            messageKey = messages("draftNoLongerAvailable.button"),
+            href = Some(config.manageTransitMovementsDraftDeparturesUrl)
+        )
```

Comment:
This was flagged up in our most recent DAC - essentially this is a link disguised as a button. What we should look to do here instead is add a POST method to the controller which does the redirect for us. See example:
- https://github.com/hmrc/manage-transit-movements-unloading-frontend/blob/1c244cbd2a44b1f1c17fc72f1ecc637b4f07ec04/app/views/UnloadingFindingsView.scala.html#L44-L46
- https://github.com/hmrc/manage-transit-movements-unloading-frontend/blob/1c244cbd2a44b1f1c17fc72f1ecc637b4f07ec04/app/controllers/UnloadingFindingsController.scala#L47-L52
-